rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+# Usecase: Extract Data from a Legacy System
+
+Extracting data from legacy systems is a common use case for RPA. 
+
+This example illustrates the following tasks:
+
+* Copy the contents from a PDF file to a text file.
+* Compare strings.
+* Copy a substring from a text file.
+* Find text on a screen using optical character recognition (OCR).
+* Interact with a screen using mouse actions and key strokes.
+* Use activity parameters to transfer data and to determine which path to follow in the model.
+
+## Example Scenario
+
+Imagine you work as an accountant at Mythical University. Mythical University has ordered a travel package for one of their field excursions. You have received an email and saved the attached order confirmation to your computer.
+
+This is what an order confirmation of your travel agency Wonderlust Travel typically looks like:
+
+[[confirmationletter-image]]
+image::usecase-1-wonderlusttravels-orderconfirmation-pdf.png["Letter containing a logo, headers, text and tables. The order ID is contained in a table in a header in the middle of the page. The label says 'Order ID'.", 75%]
+
+You need to search the order ID in your order tracking system to get the corresponding tracking number, which you need to reference in your payment.
+
+Unfortunately your order tracking system is very old and accountants can only access it by typing in and reading from a terminal screen:
+
+[[trackingsystem-allentries-image]]
+image::as400ordertrackingscreen.png["Legacy order tracking system showing green glowing text in a terminal. Above a list of orders, there is an entry field for the order id labeled 'Order-ID' and a button labeled 'Search' next to it.", 75%]
+
+You click the search field, enter the order ID, and click 'Search'.
+
+[[trackingsystem-noentries-image]]
+image::as400ordertrackingscreen-searchresult-ordernotfound.png["Legacy order tracking system shows no search results.", 75%]
+
+The list of orders is empty now. This could either mean that the order is'nt there or that you mistyped the number. Let's assume that the latter is the case. You try again and the list shows the corresponding entry:
+
+[[trackingsystem-resultentry-image]]
+image::as400ordertrackingscreen-searchresult-orderfound.png["Legacy order tracking system shows one search result.", 75%]
+
+Now you can copy the tracking number and reference it in your payment.
+
+Since all the scientists at Mythical University travel a lot, you have to repeat this tedious, error-prone and risky process several times per day. 
```

Comment:
```suggestion
Because the scientists at Mythical University travel frequently, you must repeat this tedious, error-prone, and risky process several times a day. 
```